src/main/java/calculator/IO/OutputService.java

```diff
@@ -0,0 +1,12 @@
+package calculator.IO;
+
+import static calculator.Message.Message.OUTPUT_MESSAGE;
+
+public class OutputService {
+    public void printSum(Long sum) {
+        if (sum >= 0) {
+            // 덧셈 연산 결과 출력
+            System.out.println(OUTPUT_MESSAGE + sum);
+        }
```

Comment:
단순한 궁금증입니다.
sum이 마이너스가 될 경우에 대한 처리는 일부러 따로 안 하신걸까요?

합이 마이너스가 되면 출력되는 것도 없고 예외도 터지지 않도록 되는 건가요?

Reply:
CalculatorService에서 합이 음수가 되는 경우 무조건 오버플로우 예외를 던져 "문자열 덧셈 중 오버플로우가 발생했습니다." 메시지를 출력하고 종료되게끔 의도해 출력부에 도달했을 땐 합이 음수가 되는 경우가 없을 것이라 판단해 위와 같은 코드를 작성했습니다:)

Reply:
아하 그러면 `sum`에 대한 검증은 이미 그 전 단계에서 완료한 상태이니 여기서 한 번 더 양수임을 체크하는 로직은 빼보시는 것은 어떠실까요?

```suggestion
    public void printSum(Long sum) {
        System.out.println(OUTPUT_MESSAGE + sum);

```